### DISCUSSION TITLE
[Auth] Allow a portable version with data files in the app's directory

### DISCUSSION THREAD

**Discussion by borque** [ente-io/ente#1957] [Enhancements]
Hello!
I noticed that even the ZIPped version of Ente Auth (the one that does not need to be installed) stores its data in the user's homedir.
May I suggest to store the local files in the app's directory instead and leave the homedir untouched, which would allow portable usage when combined with enhanced app locking and encryption of the local data!?
Thanks!

**Comment by nathfavour** [comment]
yeah, very good idea. There might be a way to do this tweaking yourself tho.

**Comment by tgquan67** [comment]
+1 this. In fact when I downloaded the zip package I totally expected the data to be written to the current folder, not AppData. So far I don't see any configuration file in the zip archive, so I guess this will need to be supported by the devs.
